resources/views/teams/requests.blade.php

```diff
@@ -10,9 +10,23 @@
 
     <div>
         <div class="max-w-7xl mx-auto py-10 sm:px-6 lg:px-8">
-            @livewire('booth-request', ['team' => $team])
-            <x-section-border/>
-            @livewire('sponsorship-request', ['team' => $team, 'tiers' => $tiers])
+            @if($team->is_approved)
+                @livewire('booth-request', ['team' => $team])
+                <x-section-border/>
+                @livewire('sponsorship-request', ['team' => $team, 'tiers' => $tiers])
+            @else
+                <p class="text-md text-gray-900 dark:text-gray-200">
+                    You currently cannot request a booth or sponsorship since your company is waiting for approval.<br>
```

Comment:
I don't really mind the `<br>` tags, but you can change them to `<p>` or `<div>`. Or just leave them be